scripts/lighthouse/index.mjs

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const stoplight = res => (res >= 90 ? '🟢' : res >= 75 ? '🟠' : '🔴');
+const normalizeScore = res => Math.round(res * 100);
+const formatScore = res => {
+  const normalizedScore = normalizeScore(res);
+  return `${stoplight(normalizedScore)} ${normalizedScore}`;
+};
+
+/**
+ * `core` is in scope from https://github.com/actions/github-script
+ */
+export const formatLighthouseResults = ({ core }) => {
+  // this will be the shape of https://github.com/treosh/lighthouse-ci-action#manifest
+  const results = JSON.parse(process.env.LIGHTHOUSE_RESULT);
```

Comment:
What if JSON invalid? Or variable not set? Shouldn't we error then?

Reply:
the action will fail - either way it's non blocking and would prompt us to investigate